tests/data/nexus/Ref_nexus_test.log

```diff
@@ -296,7 +296,7 @@ DEBUG - classes:
 NXdata.nxdl.xml:/AXISNAME
 DEBUG - @units - REQUIRED, but undefined unit category
 DEBUG - ===== FIELD (//entry/data/data): <HDF5 dataset "data": shape (80, 146, 195), type "<f4">
-DEBUG - value: [[0.0000000e+00 0.0000000e+00 0.0000000e+00 0.0000000e+00 0.0000000e+00 ...
+DEBUG - value: [[0. 0. 0. ... 0. 0. 0.] ...
```

Comment:
I remember, it was a CI/CD problem with this (probably different version of the library printing)

Reply:
At this moment the generated nexus_test.log is just generating '0' value without fraction part.